src/components/pages/LandingPage/LandingPage.css

```diff
@@ -1,7 +1,7 @@
 /*Section 1*/
 .landInfo1 {
   display: flex;
-  height: 600px;
+  height: 550px;
```

Comment:
This kind of detail is great to see!  Why use purely fixed units on the info div height?  Would it make sense to use vw? I do see that it is already the norm.

Reply:
For height, since we're using images, we have to use pixels unfortunately. If you can think of a way to make it work, do let me know!